rotkehlchen/db/arbitrum_one_tx.py

```diff
@@ -42,4 +42,8 @@ def _build_evm_transaction(self, result: tuple[Any, ...]) -> ArbitrumOneTransact
             nonce=result[11],
             tx_type=tx_receipt.tx_type,
             db_id=result[12],
+            authorization_list=None if len(authorization_list_result) == 0 else [
+                EvmTransactionAuthorization(nonce=entry[0], delegated_address=entry[1])
+                for entry in authorization_list_result
+            ],
```

Comment:
why receive a list instead of a nullable list? `list | None`. Does it come from the db query? If you do LEFT join doesn't it come as None here?

Reply:
> Does it come from the db query?

yes, it does. it comes as [(None, None), ...] since a transaction can have > 1 authorization.